### PR TITLE
sir i have removed adress as you told plz review

### PR DESCRIPTION
**Pull request by RitikBhardwaj7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,17 +98,17 @@ <h2>Get In Touch</h2>
 								<ul class="labeled-icons">
 									<li>
 										<h3 class="icon solid fa-home"><span class="label">Address</span></h3>
```
Comment (LieutenantR) [inline index.html:100]: Remove the icon as well

File: index.html [hunk 2]
```diff
@@ -98,17 +98,17 @@ <h2>Get In Touch</h2>
...
-										1234 Somewhere Rd.<br />
-										Nashville, TN 00000<br />
-										United States
+									
+								
+							
 									</li>
 									<li>
 										<h3 class="icon solid fa-mobile-alt"><span class="label">Phone</span></h3>
```
Comment (LieutenantR) [inline index.html:106]: Remove this icon as well.

File: index.html [hunk 3]
```diff
@@ -98,17 +98,17 @@ <h2>Get In Touch</h2>
...
-										000-000-0000
+										
 									</li>
 									<li>
 										<h3 class="icon solid fa-envelope"><span class="label">Email</span></h3>
```
Comment (LieutenantR) [inline index.html:110]: Also remove this icon

Comment (RitikBhardwaj7) [reply index.html]: done plz review

File: index.html (hunk 2 above)
Comment (RitikBhardwaj7) [reply index.html]: done
plz review

**Pull request merged**